proxy/config/ocp.go

```
@@ -0,0 +1,24 @@
+type OcpConfigHandler struct{}
+
+func (c *OcpConfigHandler) GetConfig(w http.ResponseWriter, r *http.Request) {
+	ocpConfig := OcpConfig{
+		RBACNs: RBACNs,
+	}
+	resp, err := json.Marshal(ocpConfig)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	w.Write(resp)
```

golangci-lint (1.62.2)

Error return value of `w.Write` is not checked

(errcheck)